docs/en/observability/user-experience.asciidoc

```diff
@@ -0,0 +1,194 @@
+all of which can have a greater impact on how your application performs on end-users' machines.
+
+[role="screenshot"]
+image::images/user-experience-tab.png[User experience tab]
+
+[discrete]
+[[why-user-experience]]
+== Why is {user-experience} important?
+
+Search engines are placing an increasing importance on {user-experience} when organically ranking websites.
+Elastic makes it easy to view your website data in the context of Google's Core Web Vitals --
+metrics that score three key areas of user experience: loading performance, visual stability, and interactivity.
+These Core Web Vitals are set to become the main performance measurement in Google's ranking factors.
+If you’re a content-based site that wants to appear in the “Top Stories” section of Google’s search results,
+you must have good Core Web Vitals.
+
+// We don't support business outcome capture yet. For now, this section should focus on CWV.
+// Saving this, as it might be useful later:
+// --------------------------------------------------------------------------------------------------------------
+// Every website has goals -- some sites want users to buy a product, sign up for a mailing list, download an app,
+// or share something on social media.
+// But no matter how great your product is, a poor {user-experience} can negatively impact your goal completion rate.
+// For example, in one study, 40% of users said they abandon a website if it takes more than three seconds to load.
+// footnote:[Source and more info: https://neilpatel.com/blog/loading-time/[neilpatel.com]]
+// In another, Amazon calculated that a page load slowdown of just one second would cut conversions by
+// 7% -- costing them $1.6B in sales each year.
+// footnote:[Source and more info: https://www.fastcompany.com/1825005/how-one-second-could-cost-amazon-16-billion-sales[fastcompany.com]]
+// In short, a good {user-experience} keeps your users happy and improves your website's odds of success.
+// --------------------------------------------------------------------------------------------------------------
+
+[discrete]
+[[how-user-experience-works]]
+== How does {user-experience} work?
+
+{user-experience} metrics are powered by the {apm-rum-ref}[APM Real User Monitoring (RUM) agent].
+The RUM agent uses browser timing APIs, like https://w3c.github.io/navigation-timing/[Navigation Timing],
+https://w3c.github.io/resource-timing/[Resource Timing], https://w3c.github.io/paint-timing/[Paint Timing],
+and https://w3c.github.io/user-timing/[User Timing], to capture {user-experience}
+metrics every time a user hits one of your pages.
+This data is stored in {es}, where it can be visualized using {kib}.
+
+The RUM agent can be installed as a dependency to your application, or with just a few lines of JavaScript.
+It only takes a few minutes to <<instrument-apps,get started>>.
+
+[discrete]
+[[user-experience-tab]]
+== {user-experience} in {kib}
```

Comment:
Maybe this could be a new nested page? Monitor page load durations?

Reply:
Looks like this has been addressed in this [PR](https://github.com/elastic/observability-docs/pull/198).